src/realizations/catchment/Tshirt_Realization.cpp

```diff
@@ -3,6 +3,8 @@
 #include "TshirtErrorCodes.h"
 #include "Catchment_Formulation.hpp"
 using namespace realization;
+using data_access::MEAN;
```

Comment:
This one isn't actually used, right? And for `SUM`, I would skip the `using` and just use the namespaced name below in the one call it appears in.

Reply:
`Mean `is not used here. Will make the suggested change for SUM.